scripts/issue/1139/acl_reorg.py

```diff
@@ -0,0 +1,198 @@
+
+    reward_logger = ops_multisig_old.contract(r.rewardsLogger)
+    reward_logger.grantRole(DEFAULT_ADMIN_ROLE, dev_msig)
+
+    if not sim:
+        ops_multisig_old.post_safe_tx(call_trace=True)
+
+
+# NOTE: replace your address here with rights to post tx into dev_multisig_deprecated
+ACCOUNT_NAME = ""
+
+
+def deprecated_dev_granting(sim=False):
+    if sim:
+        dev = accounts.at(r.badger_wallets.ops_deployer4, force=True)
+    else:
+        dev = accounts.load(ACCOUNT_NAME)
+
+    safe = interface.IMultisigWalletWithDailyLimit(
+        r.badger_wallets.dev_multisig_deprecated
+    )
+
+    registry = Contract(r.registry_v2)
+    guardian = Contract(registry.get("guardian"))
+
+    calldata_grant_role = guardian.grantRole.encode_input(DEFAULT_ADMIN_ROLE, dev_msig)
+
+    safe.submitTransaction(guardian, 0, calldata_grant_role, {"from": dev})
+
+    if sim:
+        owners = safe.getOwners()[0:2]
+        for owner in owners:
+            signer = accounts.at(owner, force=True)
+            safe.confirmTransaction(safe.transactionCount() - 1, {"from": signer})
+
+    C.print(
+        f"Calldata for granting ADMIN_ROLE to dev_msig=[blue]{calldata_grant_role}[/blue]. Target:[blue]{guardian.address}[/blue]\n"
+    )
+
+
+def main(safe_ens, sim=False):
+    # NOTE: before heading towards configuration, ensure current techops signers are in gas st for `MAINNET`
+    if chain.id == 1:
+        # watchlist in gas st
+        gas_st_watchlist = Contract(r.badger_wallets.gas_station).getWatchList()
+        for signer in techops_signers:
+            assert signer is gas_st_watchlist, f"Signer {signer} not in watchlist!"
+
+    target_config, safe = (
+        (TARGET_CONFIG_DEV_MSIG, dev_msig)
+        if "dev.badgerdao.eth" == safe_ens
+        else (TARGET_CONFIG_TECHOPS, techops)
+    )
+
+    registry = safe.contract(r.registry_v2, Interface=interface.IBadgerRegistryV2)
+
+    for contract_name in target_config:
+        target = safe.contract(registry.get(contract_name), from_explorer=True)
+        C.print(f"Processing configuration for {contract_name} at {target}\n")
+
+        roles_targets = target_config[contract_name]
+
+        for role in roles_targets:
+            expected_addrs_per_setting = target_config[contract_name][role]
+
+            if not isinstance(expected_addrs_per_setting, str):
+                count = target.getRoleMemberCount(role)
+                with multicall:
+                    members = [target.getRoleMember(role, i) for i in range(count)]
+
+                for addr in expected_addrs_per_setting:
+                    if not target.hasRole(role, addr):
+                        C.print(f"Grant role {role} in {contract_name} for {addr}\n")
+                        target.grantRole(role, addr)
+
+                C.print(
+                    f"Role {role} in {contract_name} has {count} members, checking against target config...\n"
+                )
```

Comment:
Nit but I suggest that you move this to before the role granting block, it would make the output a lot more clear to interpret.

Reply:
move above the code block in dc7554ef6fcfcd4614b4936bd7dbdba67573ebd9